go.mod

```diff
@@ -9,7 +9,7 @@ require (
 	github.com/ktrysmt/go-bitbucket v0.6.2
 	github.com/onsi/ginkgo v1.14.0
 	github.com/onsi/gomega v1.10.1
-	github.com/xanzy/go-gitlab v0.33.0
+	github.com/xanzy/go-gitlab v0.43.0
```

Comment:
This change should go in its own commit, perhaps in its own PR.

Reply:
that is in master already, maybe I did something wrong when I ran `git merge master` ? or the squash ? 🤔